### PR TITLE
RGB Poggit 🌈

### PR DESCRIPTION
**Pull request by HBIDamian**
RGB Commonly stands for "Red Green Blue".
I tend to think it stands for "Really Great Brain". It's a RGB Move here bois!

### CODE REVIEW

**Comment by SOF3** [comment]
Meanwhile almost everything else in Poggit is B&W...

**Comment by HBIDamian** [comment]
PocketMine, bringing colour to your plugin repo one commit at a time!

**Review by NhanAZ — APPROVED** [review]

**Review by 0xliam627 — APPROVED** [review]

File: res/legacy-poggit-icon.ico
Comment (JaxkDev) [inline res/legacy-poggit-icon.ico:1]: Legacy doubled in size?

File: src/poggit/module/HtmlModule.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ protected function headIncludes(string $title, $description = "", $type = "websi
       <meta name="keywords"
             content="<?= implode(",", array_merge([Mbd::esq($title)], $extraKeywords)) ?>,plugin,PocketMine,pocketmine plugins,MCPE plugins,Poggit,PocketMine-MP,PMMP"/>
       <meta property="og:site_name" content="Poggit"/>
-      <meta property="og:image" content="<?= $image ?? (Meta::getSecret("meta.extPath") . (date("M j") === "Apr 1" ? "res/mrs-poggit.png" : "res/poggit-icon.png")) ?>"/>
+      <meta property="og:image" content="<?= $image ?? (Meta::getSecret("meta.extPath") . (date("M j") === "Apr 1" ? "res/mrs-poggit.png" : "res/poggit-icon.gif")) ?>"/>
```
Comment (JaxkDev) [inline src/poggit/module/HtmlModule.php:44]: See L56 too.

**Review by JaxkDev — CHANGES_REQUESTED** [review]
Any chance of this getting compressed (if any more) to a lower size too?

Dylan mentioned getting the pmmp size slashed by 50%.

**Comment by HBIDamian** [comment]
> Any chance of this getting compressed (if any more) to a lower size too?
> 
> Dylan mentioned getting the pmmp size slashed by 50%.

The size is the same size as the legacy poggit icon. Any smaller, and it's ugly as heck. All I did with the legacy one was rename it. No resize was done. 

As far as compression, I'll compress the gif when I get home. The downside with compression is that it really makes the gif look terrible.

File: src/poggit/module/HtmlModule.php (hunk 1 above)
Comment (CelticTrinculo) [reply src/poggit/module/HtmlModule.php:44]: > See L56 too.

Already covered by https://github.com/HBIDamian/poggit/blob/beta/res/poggit-icon.ico :P Step ahead of you, bruv! :D

Comment (JaxkDev) [reply src/poggit/module/HtmlModule.php:44]: My bad, too hot to think here and saw the red decrease in file size as a deleted file 🤦 (hence the doubled legacy size confusion, sorry).

**Review by JaxkDev — APPROVED** [review]

**Comment by JaxkDev** [comment]
@SOF3 if you're happy with size as-is, it's good to go

**Comment by dktapps** [comment]
ezgif com-optimize

ezgif lossy compression with level 5, looks basically the same but with half the filesize

**Comment by JaxkDev** [comment]
> ezgif com-optimize
> 
> ezgif lossy compression with level 5, looks basically the same but with half the filesize

Thats coming up over a mb for me dl from github lmao

**Comment by HBIDamian** [comment]
@dktapps Isn't that one from Discord? 
The one I uploaded in the PR [here](https://raw.githubusercontent.com/poggit/poggit/7f3b503668a466fb12a99269f12447b0c969b6c2/res/poggit-icon.gif) is 128px^2. That ones 512px^2

**Comment by dktapps** [comment]
My bad, I assumed it was the same.

**Comment by SOF3** [comment]
I don't have a problem with the file size, I just think the fanciness of the color really doesn't fit with the simplicity of the color scheme in the rest of the site.

**Comment by JaxkDev** [comment]
Reverted as requested.

**Comment by HBIDamian** [comment]
Where's my RGB army at? 😂

**Comment by NhanAZ** [comment]
> Where's my RGB army at? 😂

A denial of service attack with 50 million requests per second will be deployed☹️

**Comment by HBIDamian** [comment]
> > Where's my RGB army at? 😂
> 
> A denial of service attack with 50 million requests per second will be deployed☹️

I can't endorse that, as much as I dislike Mrs Poggit... 😅

**Comment by inxomnyaa** [comment]
Definitely support this, the readability is much improved with the black git icon on there

**Review by inxomnyaa — COMMENTED** [review]
Looks good, ~200kb size difference is not too bad either

**Comment by HBIDamian** [comment]
The benefits outweigh the negatives

Benefits:
- Its RGB, Roasting Great Britain!
- a fresh logo with effort into it as the old one is just shit. Sorry, but it's because incredibly low effort. 
\-\_\- 
- easier to see at a glance. The nice&bold black git logo stands out. Previously it was a blehhh on the PM logo
- Removed the "PM" in the background for easier visibility
- finally brings some life into poggit, instead of it just being a plain boring mess. 

Negatives:
- Sofe doesn't like it
Thats it. Only negative.

**Comment by SOF3** [comment]
Rejected due to the disharmony of the gradient color scheme from the rest of the site.

**Comment by HBIDamian** [comment]
The old one was disharmony too. Was very much out of place, and it was disharmony to anyone with eyes.

**Comment by SOF3** [comment]
The problem with Poggit's UX has nothing to do with its colors and icons. It's to do with all the HTML + AJAX requests stacking up the latency, ancient jQuery scripts and extreme NIH components.

**Comment by inxomnyaa** [comment]
Didnt even notice the removal of the PM, that might be what actually helps the readability

**Comment by SOF3** [comment]
Yeah, definitely, improve readability by making there nothing to read.

**Comment by NhanAZ** [comment]
I think this logo matches Poggit's color scheme
poggit-icon
I guess SOF3 likes this logo too

**Comment by HBIDamian** [comment]
He's gonna have to because it already matches poggits colourless and lifeless theme. 
Well played, good sir!

**Comment by dktapps** [comment]
Guys, just let it lie.

@SOF3 is the authority here, if he doesn't want to change it, that's up to him. This is, after all, his project.

**Pull request merged**